src/components/AoRadio.vue

```diff
@@ -5,11 +5,14 @@
       v-bind="$attrs"
       :value="val"
       type="radio"
+      class="ao-checkbox__input"
```

Comment:
should this class and the other one be ao-radio__x since the component is radio

Reply:
yup,
 ill do the fixups for this PR

Reply:
fixed